contrib/plugins/cowsay.sh

```diff
@@ -0,0 +1,25 @@
+#! /bin/sh
+
+# Eg. {"jsonrpc":"2.0","id":2,"method":"getmanifest","params":{}}\n\n
+read JSON
+read BLANK
+id=$(echo "$JSON" | sed 's/.*"id" *: *\([0-9]*\),.*/\1/')
+
+echo '{"jsonrpc":"2.0","id":'$id',"result":{"options":[],"rpcmethods":[{"name":"cowsay","usage":"<string>","description":"Have a cow, man!"}]}}'
+
+# Eg. {"jsonrpc":"2.0","id":5,"method":"init","params":{"options":{},"configuration":{"lightning-dir":"/home/rusty/.lightning","rpc-file":"lightning-rpc","startup":false}}}\n\n
+read JSON
```

Comment:
Consider doing `read -r` instead of `read` to avoid mangling backslashes. Plus we'll get stripping of leading/trailing spaces for free! Applies to all `read` in this script.